### PR TITLE
Gameplay Tweaks

### PR DESCRIPTION
**Pull request by LoganTHarvell**
Changed the way planet resources are updated when a player is docked. Also updated the game screen, specifically in regards to time and score. Lastly, slightly lengthened the growth period to slow population changes.

### CODE REVIEW

File: Astral Pathfinder/Models/Planet.hpp [hunk 1]
```diff
@@ -39,13 +39,13 @@ namespace PlanetParameters {
   const int homeStartPopulation = 5000;
   const int homeStartFertility = 100;
   const int homeStartDeposits = 250;
-  const int homeStartMiningPercent = 50;
+  const int homeStartMiningPercent = 58;
```
Comment (holdemani) [inline Astral Pathfinder/Models/Planet.hpp:42]: Hmmm? What caused this change?

File: Astral Pathfinder/Models/Planet.hpp [hunk 2]
```diff
@@ -39,13 +39,13 @@ namespace PlanetParameters {
...
   const int homeStartFarmingPercent = 100 - homeStartMiningPercent;
   const int homeStartInfraPercent = 0;
   const int homeStartReservePercent = 100 - homeStartInfraPercent;
   
   // Defines population growth information
-  const int growthPeriod = 900;      // Frames per growth period
+  const int growthPeriod = 1200;      // Frames per growth period
```
Comment (holdemani) [inline Astral Pathfinder/Models/Planet.hpp:48]: Why would you make this longer? it already felt long

**Comment by holdemani** [comment]
Does this complete your gameplay tweaks issue? If so, you should update it

File: Astral Pathfinder/Models/Planet.hpp (hunk 2 above)
Comment (holdemani) [reply Astral Pathfinder/Models/Planet.hpp:48]: Just tested the game. When going to a new planet, the population doesn't begin to grow until what I assume is a new growthPeriod, and that takes a while. Is this intentional?

File: Astral Pathfinder/Models/Planet.hpp (hunk 1 above)
Comment (LoganTHarvell) [reply Astral Pathfinder/Models/Planet.hpp:42]: It was changed due to Ship population causing overproduction

File: Astral Pathfinder/Models/Planet.hpp (hunk 2 above)
Comment (LoganTHarvell) [reply Astral Pathfinder/Models/Planet.hpp:48]: That was just a coincidence, also it worked better longer cause it makes the rate of change slower

Comment (LoganTHarvell) [reply Astral Pathfinder/Models/Planet.hpp:48]: Newly changed to reset growth rate when player docks, but also prevents multiple resets by redocking.

**Comment by holdemani** [comment]
Much better, thank you

**Pull request merged**